scipy/stats/_stats_py.py

```diff
@@ -7501,48 +7501,84 @@ def chisquare(f_obs, f_exp=None, ddof=0, axis=0):
            in the case of a correlated system of variables is such that it can be reasonably
            supposed to have arisen from random sampling", Philosophical Magazine. Series 5. 50
            (1900), pp. 157-175.
+    .. [4] Mannan, R. William and E. Charles. Meslow. “Bird populations and
+           vegetation characteristics in managed and old-growth forests,
+           northeastern Oregon.” Journal of Wildlife Management
+           48, 1219-1238, :doi:`10.2307/3801783`, 1984.
 
     Examples
     --------
-    When just `f_obs` is given, it is assumed that the expected frequencies
-    are uniform and given by the mean of the observed frequencies.
+    In [4]_ bird foraging behavior was investigated in a forest of Oregon.
+    The foraging refers to the range of activities and behaviours exhibited by
+    birds in their quest for food.
+    In an old-growth forest, 44% of the canopy volume was Douglas fir,
+    24% was ponderosa pine, 29% was grand fir, and 3% was western larch.
+    They observed the behavior of several species of birds, one of which
+    was the red-breasted nuthatches. They made 189 observations of foraging
+    of this specie with the respective frequencies of 23%, 27%, 29% and 21%.
```

Comment:
```suggestion
    of this species with the respective frequencies of 23%, 27%, 29%, and 21%.
```

That's just a grammatical correction. It could still be more precise.

Based on the fact that there are four trees, I infer that these percentages have something to do with the four trees, but the relationship between foraging and species of trees should be defined. Is this the percentage of birds that visited that type of tree, or is it the percentage of existing trees that birds visited, or the percentage of bird foraging events that occurred in the type of tree? (Probably the last?)